csharp/ql/src/Telemetry/ExternalApi.qll

```diff
@@ -24,14 +24,28 @@ class TestLibrary extends RefType {
   }
 }
 
+/** Holds if the given callable is a constructor without parameters. */
+private predicate isParameterlessConstructor(DotNet::Callable c) {
+  c instanceof Constructor and c.getNumberOfParameters() = 0
+}
+
+/** Holds if the given callable is part of a common testing library or framework. */
+private predicate isTestLibrary(DotNet::Callable c) { c.getDeclaringType() instanceof TestLibrary }
+
+/** Holds if the given callable is not worth supporting. */
+private predicate isUninteresting(DotNet::Callable c) {
+  isTestLibrary(c) or isParameterlessConstructor(c)
```

Comment:
Inline isTestLibrary and isParameterlessConstructor.

Reply:
Dumb question, but just to be sure: Is this a style suggestion to move `isParameterlessConstructor(c)` underneath `isTestLibrary(c)`? Or are you asking me to use [pragma[inline]](https://codeql.github.com/docs/ql-language-reference/annotations/#pragma-inline) for these?

Reply:
ah, that makes a lot more sense, thanks! Updated in https://github.com/github/codeql/pull/11631/commits/a889cc4f6555e907debb4f23aefa631a43280c97.